packages/discovery-provider/src/api/v1/tracks.py

```diff
@@ -1632,6 +1632,7 @@ class BestNewReleases(Resource):
         id="Best New Releases",
         description='Gets the tracks found on the "Best New Releases" smart playlist',
     )
+    @full_ns.expect(best_new_releases_parser)
```

Comment:
Do the calls already work against staging/prod without this parser decorator? Just wondering if we have to do a DN release before client goes out to keep from breaking things.

Reply:
oh good call, no they don't. i think we're good tho cuz the protocol release will go out on monday, and client won't go out until tues earliest?